cmd/kyma/alpha/sign/module/module.go

```diff
@@ -0,0 +1,87 @@
+package module
+
+import (
+	"fmt"
+	"github.com/gardener/component-spec/bindings-go/ctf"
+	"github.com/kyma-project/cli/internal/cli"
+	"github.com/kyma-project/cli/pkg/module"
+	"github.com/mandelsoft/vfs/pkg/osfs"
+	"github.com/spf13/cobra"
+)
+
+type command struct {
+	opts *Options
+	cli.Command
+}
+
+//NewCmd creates a new Kyma CLI command
+func NewCmd(o *Options) *cobra.Command {
+
+	c := command{
+		Command: cli.Command{Options: o.Options},
+		opts:    o,
+	}
+
+	cmd := &cobra.Command{
+		Use:   "module MODULE_NAME MODULE_VERSION [flags]",
+		Short: "Sign all module resources from unsigned component descriptor which hosted in a remote OCI registry",
+		Long: `Use this command to sign a Kyma module.
+
+### Detailed description
+
+This command signing all module resources recursively based on an unsigned component descriptor which hosted in a remote OCI registry with provided private key, the output (signed-component-descriptor.yaml) will be saved in the descriptor path (./mod as a default) as 
```

Comment:
```suggestion
This command signs all module resources recursively based on an unsigned component descriptor hosted in an OCI registry with the provided private key. The output (signed-component-descriptor.yaml), is then saved in the descriptor path (./mod as a default) as 
```

This sentence seems to be not finished, what should go after "saved in the descriptor path (./mod as a default) as"?